identity-account-storage/src/storage/stronghold.rs

```diff
@@ -674,6 +747,12 @@ pub(crate) fn random_location(key_type: KeyType) -> KeyLocation {
   KeyLocation::new(key_type, fragment, &public_key)
 }
 
+pub(crate) fn random_stronghold_location() -> Location {
```

Comment:
Nit: maybe just leave the function private unless otherwise necessary, only used here.

```suggestion
fn random_stronghold_location() -> Location {
```